zipline/__main__.py

```diff
@@ -51,6 +60,8 @@ def main(extension, strict_extensions, default_extension):
         os.environ,
     )
 
+    create_args(x, zipline.extension_args)
```

Comment:
should we parse the args before loading the extension? I could imagine the extension code wanting access to this to find resources